index.bs

```diff
@@ -2680,8 +2731,15 @@ attributes.
 
 <div dfn-type="dict-member" dfn-for="AuthenticatorSelectionCriteria">
     :   <dfn>authenticatorAttachment</dfn>
-    ::  If this member is present, eligible authenticators are filtered to only authenticators attached with the
-        specified [[#enum-attachment]]. The value SHOULD be a member of {{AuthenticatorAttachment}} but [=client platforms=] MUST ignore unknown values, treating an unknown value as if the [=map/exist|member does not exist=].
+    ::  If this member is present, eligible [=authenticators=] are filtered to be only those authenticators attached with the specified
+        [[#enum-attachment|authenticator attachment modality]] (see also [[#sctn-authenticator-attachment-modality]]).
+        If this member is absent, then the [=[RP]=] is saying that any attachment modality is acceptable.
+        The value SHOULD be a member of {{AuthenticatorAttachment}} but [=client platforms=] MUST ignore unknown values,
+        treating an unknown value as if the [=map/exist|member does not exist=].
+
+        The [=[RP]=] can determine the resulting [=authenticator attachment modality|attachment=] for the created credential
```

Comment:
Also a good point. I don't think we really clarify _how_ one can determine the authenticator attachment(s) from the `transports` list, so it might be appropriate to introduce a small section on that. We already have [§5.2.1.1. Easily accessing credential data](https://www.w3.org/TR/2021/REC-webauthn-2-20210408/#sctn-public-key-easy) as a subsection of the `AuthenticatorAttestationResponse` section, so another sibling subsection there seems reasonable. I think I'll do that as a meta-pull request into this one.

Reply:
Addressed in meta-PR #1670.

Reply:
It is good that #1670 adds an explanation of how `getTransports()` may be interpreted by the RP. That said, I don't think it addresses the original comment here, at least not always. If `getTransports()` returned a list, with one of those values being `internal`, and another being a different transport (such as the mobile phone as an authenticator use case), the RP cannot determine the resulting authenticator attachment modality for the created credential. This however *could* be determined with the proposal in #1668, which I think better addresses the use case of an RP trying to determine if a platform-bound credential has been created as part of this ceremony.

I am in favour of keeping the changes in #1670 to aid in the understanding of RPs interpretation and use of `getTransports()`, but I don't think that this section should attempt to claim that the `getTransports()` API is alone enough to determine the resulting authenticator attachment modality.

Reply:
Oh right, you had already proposed that in https://github.com/w3c/webauthn/pull/1668#discussion_r704924376 . :slightly_smiling_face:

Reply:
To land this PR, we could just remove this section completely:
```
The [=[RP]=] can determine the resulting [=authenticator attachment modality|attachment=] for the created credential using the {{AuthenticatorAttestationResponse/getTransports()}} method of the resulting {{AuthenticatorAttestationResponse}}.
   See [[#enum-transport]] for details.
```

and let #1668 deal with the issue of deterministically allowing an RP to understand the attachment modality of an authenticator for a particular ceremony.

Reply:
Ok, yeah, that seems like a fair middle ground. We can keep #1670 open for tracking that we should add this back in once we have #1668 sorted.

Reply:
Yeah, that works too I guess. There's not really much reason to hurry to get this one merged.

Reply:
This thread is now resolved given @emlun's having "...updated this to align with the new `authenticatorAttachment` property" ?